src/vivarium/framework/randomness.py

```diff
@@ -157,7 +158,9 @@ def convert_to_ten_digit_int(self, column: pd.Series) -> pd.Series:
             numeric.
 
         """
-        if isinstance(column.iloc[0], datetime.datetime):
+        if len(column) == 0:
```

Comment:
Under what circumstances do you get here with an empty series?

Reply:
The only call is from `update`, which should short-circuit, so we don't need/want the extra branch here. as a non-empty series is a precondition for calling.

Reply:
We get that when we have an empty population.

Reply:
Hmm I was reaching this with the empty population, but now I'm not. Looking at the code I agree it should short-circuit in `update`.